cmake/linux/apprun-hooks/jack-hook.sh

```diff
@@ -1,15 +1,11 @@
-#!/usr/bin/env bash
+#!/bin/sh
 
 # Workaround crash when jack is missing by providing a dummy version
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
-ME="$( basename "${BASH_SOURCE[0]}")"
-# Set language to English
-export LC_ALL=C
-if ldd "$DIR/usr/bin/lmms" |grep "libjack.so" |grep "not found" > /dev/null 2>&1; then
-	echo "[$ME] Jack does not appear to be installed.  That's OK, we'll use a dummy version instead." >&2
-	export LD_LIBRARY_PATH="$DIR/usr/lib/jack:$LD_LIBRARY_PATH"
+export LC_ALL=C # Set language to English
+if ldd "$APPDIR/usr/bin/lmms" |grep "libjack.so" |grep "not found" > /dev/null 2>&1; then
```

Comment:
No need to set/reset if you use
```suggestion
if LC_ALL=C ldd "$APPDIR/usr/bin/lmms" |grep "libjack.so" |grep "not found" > /dev/null 2>&1; then
```